ui/pages/login/organizations.js

```diff
@@ -28,6 +28,8 @@ export default function Organizations() {
             key={o.url}
             className='my-2 inline-flex w-full items-center justify-between rounded-md border border-gray-300 bg-white py-2.5 px-4 text-xs text-gray-500 shadow-sm hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-blue-500 focus:ring-offset-2'
           >
+            {o.name}
```

Comment:
lets style this a little bit
maybe we can convert this whole `<a />` to an `<button />`
something like this - you might want to play around a bit though 
`<button
                onClick={...}
                key={o.url}
                title={`${o.name} — ${o.url}`}
                className='my-2 inline-flex w-full items-center rounded-md border border-gray-300 bg-white py-2.5 px-4 shadow-sm hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-blue-500 focus:ring-offset-2'
              >
                <div className='text-xs text-gray-500'>{o.name}</div>
                <div className='text-2xs text-gray-400'>{o.url}</div>
              </button>`

Reply:
Excited to see what this looks like